app/redux/actions/unlockWalletAction.ts

```diff
@@ -14,6 +14,7 @@ export type UnlockWalletAction = {
     selectedAccount: SelectedAccount;
     walletStatus: WalletStatus;
     offlineModeEnabled: boolean;
+    accounts: Accounts;
```

Comment:
set accounts on wallet unlock. This makes sure we can open the wallet with an account selected